sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/Upload.java

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * The Upload object can accept byte chunks in the form of Parts.
+ */
+@Immutable
+public final class Upload implements JsonSerializable<Upload> {
+
+    /*
+     * The Upload unique identifier, which can be referenced in API endpoints.
+     */
+    @Generated
+    private final String id;
+
+    /*
+     * The Unix timestamp (in seconds) for when the Upload was created.
+     */
+    @Generated
+    private final long createdAt;
+
+    /*
+     * The name of the file to be uploaded.
+     */
+    @Generated
+    private final String filename;
+
+    /*
+     * The intended number of bytes to be uploaded.
+     */
+    @Generated
+    private final long bytes;
+
+    /*
+     * The intended purpose of the file. [Please refer here](/docs/api-reference/files/object#files/object-purpose) for
```

Comment:
I will take note to update this comment. the `[Please refer here]` would not lead anywhere in this context.